src/qtviewer/ParseIfcFile.cpp

```diff
@@ -0,0 +1,46 @@
+#include "ParseIfcFile.h"
+
+#include "../ifcparse/Ifc2x3.h"
+#define IfcSchema Ifc2x3
+
+#include "../ifcgeom/IfcGeom.h"
```

Comment:
```suggestion

```

This ought to be better documented in IfcOpenShell, but basically stay away from the ../ifcgeom/ folder. Only use `../ifcgeom_schema_agnostic`. That way the IfcSchema define is not necessary and the viewer can open any supported schema.